cmd/rewards.go

```diff
@@ -51,6 +52,12 @@ func init() {
 		"the default monthly choices for the rewards system")
 	must(viper.BindPFlag("default-monthly-choices", rewardsCmd.PersistentFlags().Lookup("default-monthly-choices")))
 	must(viper.BindEnv("default-monthly-choices", "DEFAULT_MONTHLY_CHOICES"))
+
+	// defaultACChoices - defaults to empty (which causes the choices to be dynamic)
+	rewardsCmd.PersistentFlags().StringVarP(&defaultACChoices, "default-ac-choices", "", ``,
```

Comment:
Any reason for the backticks instead of the quotes?

Reply:
No particular reason. Changed in https://github.com/brave-intl/bat-go/pull/415/commits/8ff6c7f5afb1cd1c2522bb3e2ab6bfb3213fafb3